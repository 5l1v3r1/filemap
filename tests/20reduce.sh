#!/bin/sh

fm -v map -i '/test/*' "`pwd`/split.sh <> sort | uniq -c"
#fm ls /reduce/*/*
