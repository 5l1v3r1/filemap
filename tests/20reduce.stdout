      2 4	5	6
      2 1	2	3
      1 1	2	8
      1 1	3	5
      2 7	8	9
/reduce/ZEChgBcFSuez0fy0yq+I9tpy7t8/1.part.d/.status
/reduce/ZEChgBcFSuez0fy0yq+I9tpy7t8/1.part.d/stdout
/reduce/ZEChgBcFSuez0fy0yq+I9tpy7t8/4.part.d/.status
/reduce/ZEChgBcFSuez0fy0yq+I9tpy7t8/4.part.d/stdout
/reduce/ZEChgBcFSuez0fy0yq+I9tpy7t8/7.part.d/.status
/reduce/ZEChgBcFSuez0fy0yq+I9tpy7t8/7.part.d/stdout
/reduce/p2MPmJzos7dko2hcszo1wPYtYD8/stdout.d/.status
/reduce/p2MPmJzos7dko2hcszo1wPYtYD8/stdout.d/stdout
/reduce/tmGi_o2yWwS0amS_2XdzGClpQbU/1.part.d/.status
/reduce/tmGi_o2yWwS0amS_2XdzGClpQbU/1.part.d/stdout
/reduce/tmGi_o2yWwS0amS_2XdzGClpQbU/1.part.d/stdout.d
/reduce/tmGi_o2yWwS0amS_2XdzGClpQbU/4.part.d/.status
/reduce/tmGi_o2yWwS0amS_2XdzGClpQbU/4.part.d/stdout
/reduce/tmGi_o2yWwS0amS_2XdzGClpQbU/4.part.d/stdout.d
/reduce/tmGi_o2yWwS0amS_2XdzGClpQbU/7.part.d/.status
/reduce/tmGi_o2yWwS0amS_2XdzGClpQbU/7.part.d/stdout
/reduce/tmGi_o2yWwS0amS_2XdzGClpQbU/7.part.d/stdout.d
